src/views/viewer/ChannelView/ChannelView.tsx

```diff
@@ -55,20 +99,98 @@ export const ChannelView: React.FC = () => {
       Logger.warn('Failed to update Channel following', { error })
     }
   }
-  if (error) {
+  if (videosError) {
+    throw videosError
+  } else if (error) {
     throw error
+  } else if (errorSearch) {
+    throw errorSearch
+  }
+
+  const handleSetCurrentTab = async (tab: number) => {
+    setIsSearching(false)
+    setCurrentVideosTab(tab)
+  }
+  const handleSorting = (value?: VideoOrderByInput | null | undefined) => {
+    if (value) {
+      setSortVideosBy(value)
+      refetch({ orderBy: value })
+    }
+  }
+  const handleOnResizeGrid = (sizes: number[]) => setVideosPerRow(sizes.length)
+  const handleChangePage = (page: number) => {
+    isSearching ? setCurrentSearchPage(page) : setCurrentPage(page)
+  }
+  const handleKeyPress = (e: React.KeyboardEvent<HTMLInputElement>) => {
```

Comment:
Meaningful name please
```suggestion
  const handleKeyPress = (event: React.KeyboardEvent<HTMLInputElement>) => {
```